cves/kernel/CVE-2019-15666.yml

```diff
@@ -320,8 +325,8 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: false
+  note: There was no discussion around this issue
```

Comment:
https://lore.kernel.org/all/20190228075226.hza3hydllddj65nd@gondor.apana.org.au/T/ one person acked, another accepted the patch